packages/terra-form-checkbox/CHANGELOG.md

```diff
@@ -3,4 +3,6 @@ ChangeLog
 
 (Unreleased)
 -----------------
-Initial stable release
+### Added
+* Import terra-mixins into style sheet
+* Initial stable release
```

Comment:
Has this been released? If not, I feel like we could tweak the changelog to just mention `Initial Release`